SETUP/UNICODE.md

```diff
@@ -42,7 +42,7 @@ that are shown to the users in the proofreading interface character picker.
 Picker sets are optional and, if they are used, need not cover the entire set
 of codepoints in the character suite.
 
-The code currently ships with 4 character suites:
+The code currently ships with 9 character suites:
```

Comment:
This will probably get out of date like before. Maybe reword to just ```The code currently ships with the following character suites:```?

Reply:
This is a great idea.

Reply:
I agree. I'm waiting on a wiki page so I can update everything at the same time.

Reply:
Done.